files/calenstyle/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
```

Comment:
There are no tags here https://github.com/CuriousSolutions/CalenStyle/releases
The bot will fail